### ISSUE TITLE
Please provide feedback for lazygit!

### ISSUE THREAD

**Issue by jesseduffield** [jesseduffield/lazygit#1220]
I've whipped up a questionnaire to help get an idea for what changes people would like to see in lazygit, and what its current strengths are. It's only a handful of questions

https://forms.gle/AdFEuLy2dikcmK4F7

Any and all feedback is much appreciated!  🙏

**Comment by Sandesh333333** [comment]
> I've whipped up a questionnaire to help get an idea for what changes people would like to see in lazygit, and what its current strengths are. It's only a handful of questions
> 
> https://forms.gle/AdFEuLy2dikcmK4F7
> 
> Any and all feedback is much appreciated! 🙏

**Comment by Hamza12700** [comment]
Is lazygit supports markdown?
when I want to commit a message can I use markdown syntax?

**Comment by mchisolm0** [comment]
Is the form still intended to be filled out? I did, but if not wanted to know where I should pitch a suggestion and gauge feasibility. Thanks in advance, and I appreciate this project SO much! 😁

**Comment by jesseduffield** [comment]
@mchisolm0 yes I still check on the form responses every now and then. But if you have a specific suggestion to pitch, please raise an issue

**Comment by gfvioli** [comment]
I'm gonna paste my comment made in another discussion [here](https://github.com/jesseduffield/lazygit/issues/3561#issuecomment-2272957528)

> Just today I ran into the same situation as you when testing bare repos on lazygit and personally I find it confusing. I think that I have a couple of suggestions to enhance the experience when working with worktrees on a bare repo:
> 
> * Ability to execute lazygit in the root of the bare repo, right now you have to go into one of the worktrees for lazygit to be able to launch. This is just a minor QoL not having to do cd once more and go to a worktree before calling lazygit.
> * I think in the case of a bare repo, worktrees currently located in panel 2 should replace local branches in panel 3. That way you avoid the confusion and all the clutter with local branches. Also allows for a speedier workflow since you don't have to be changing panel's 2 depth to see files status.
> 
> To be honest, I'm not quite sure about the rationale to put worktrees on pane 2, my understanding is that while you can do worktrees on a NOT bare repo, as per ThePrimeagen video on it (link [here](https://www.youtube.com/watch?v=2uEqYw-N8uE)), the structure is a mess. Thus I think the recommendation is to use bare repos when using worktrees such that you can use them as independent branches, which would lead to this recommendation.
> 
> I'll keep on experimenting with bare repos and I'll keep on providing feedback.

**Comment by DavidingPlus** [comment]
Looks really nice, but I am used to using basic command line and VS Code.

I really love to try and search when I got some time.

**Comment by atljoseph** [comment]
A coworker turned me into lazygit. Thanks for the awesome tool. 

A few questions:
- Is there a view in lazygit where you can view and compare information about multiple repos? Like a multi-repo explorer?
- And are there any current functions to operate on multiple repos? For example: stage, commit annd push all files among multiple repos at once. 

My use case: 
- I want to build a tool that performs both git and go commands across many repositories. Besides the obvious value that lazygit has for git workflows alone, this feels like an interesting place (to me) to start asking around. 

Thank you!
